symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/loadbalancing/ExternalLoadBalancingStrategy.java

```diff
@@ -0,0 +1,42 @@
+package com.symphony.bdk.core.client.loadbalancing;
+
+import com.symphony.bdk.core.config.model.BdkConfig;
```

Comment:
some imports can be cleaned there (you can configure idea to optimize imports for you)